web/client/utils/EncodeUtils.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Convert an UTF-8 into a byte representation. It usefult to avoid the `btoa` Unicode problem
+ * A more detailed explaination about this can be found here:
+ *
+ * @param {string} string string to convert
+ * @returns the UTF8 string converted in bytes
+ */
+export function encodeUTF8(string) {
```

Comment:
could be useful to have unit tests for this utils?

Reply:
sure. I tested it in the action but a test here is better. 
Added a test file right now